practical-07/src/components/GalleryCard.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+    <div class="edit-delete-btn">
+      <p type="button" class="btn-edit" @click="editCarDetails(carId)"><i class="fa fa-pencil"></i></p>
+      <p type="button" class="btn-delete" @click="dataDelete">x</p>  
+    </div>
+    <div class="carName">{{ carName }}</div>
+    <div class="col-auto mb-2 card-animation"><img :src="carImage" alt="car-image" /></div>
+    <div class="mb-2">
+      <button class="info-btn" size="sm" pill variant="outline" @click="carDataDisplay">
+          <i class="fa-solid fa-circle-info"></i>
+          <span class="separator"/>
+          <span class="text">View details</span>
+      </button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'GalleryCard',
+  emits: ["carDetailDisplay","carDelete"],
+  props: {
+    carId: Number,
+    carName: String,
+    carImage: String,
+    carDetails: String,
+    carPrice: String,
+    editCarDetails: Function,
+  },
+  methods: {
+    dataDelete() {
+      this.$emit("carDelete");
+    },
+    
+    carDataDisplay() {
+       this.$emit("carDetailDisplay");
+    },
+  },     
+};
+</script>
+
+<style>
+.card-animation:hover {
+transform: scale(0.9);
```

Comment:
Codacy has a fix for the issue: [Insert `··`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
  transform: scale(0.9);
```